src/workerd/server/server.c++

```diff
@@ -1392,6 +1526,25 @@ public:
       KJ_LOG(ERROR, exception);
     }
 
+    // Removes actors from `actors` after 70 seconds of last access.
+    kj::Promise<void> cleanupLoop() {
+      constexpr auto EXPIRATION = 70 * kj::SECONDS;
+
+      while (true) {
+        auto now = timer.now();
+        actors.eraseAll([&](auto&, kj::Own<ActorContainer>& entry) {
+          if (entry->hasClients()) {
+            // We are still using the actor so we cannot remove it!
+            return false;
+          }
+
+          return (now - entry->getLastAccess()) > EXPIRATION;
+        });
+
+        co_await timer.afterDelay(EXPIRATION).eagerlyEvaluate(nullptr);
```

Comment:
Is the internal code also based on doing mass evictions every 70 seconds, rather than a per-actor timer that waits 70 seconds after the last client disconnects?

I think I recall that it is, in which case this looks good -- we should match the behavior.

Reply:
Yup, they're more or less the same. This one is a bit simpler and we're using a coroutine instead.